### PR TITLE
Protocol from old rootUrl prepended to new rootUrl

### LINKED ISSUE #3: Add protocol from old rootUrl to replacement rootUrl

### PR DESCRIPTION
**Pull request by SwinX**
resolves #3

### CODE REVIEW

File: test/lib/plugin.js [hunk 1]
```diff
@@ -113,6 +116,25 @@ describe('plugin', function () {
                 expect(gemini.config.forBrowser('ya_browser').rootUrl).to.be.equal('some_host:1');
             });
         });
+
+        it('should prepemnd protocol from old root url to new root url', function  () {
```
Comment (j0tunn) [inline test/lib/plugin.js]: `should save protocol in root url`?

**Comment by arikon** [comment]
@SwinX Why don't you squash commits?

**Comment by SwinX** [comment]
Just forgot, my bad. Next time will squash

**Pull request merged**